kernel/Cargo.toml

```diff
@@ -102,8 +100,8 @@ integration-test = ["hdfs-native-object-store/integration-test"]
 arrow = ["arrow-56"] # latest arrow version
 need-arrow = [] # need-arrow is a marker that the feature needs arrow dep
 
-arrow-55 = ["dep:arrow_55", "dep:parquet_55", "object_store", "comfy-table"]
-arrow-56 = ["dep:arrow_56", "dep:parquet_56", "object_store", "comfy-table"]
+arrow-55 = ["dep:arrow_55", "dep:parquet_55", "object_store"]
```

Comment:
It hasn't been released for the 55 branch 😢

Reply:
(also surprised this passes tests? - shouldn't MSRV fail for arrow 55 target? i guess maybe we don't test MSRV specifically for 55?)

Reply:
Yes, I was hoping to get rid of the `TODO`, but it looks like we need to keep it around a bit longer. Updated the PR 👍